threads/tests.py

```diff
@@ -394,3 +418,56 @@ def test_replace_nonexisting_user(self) -> None:
             tasks.replace_user,
             {"old": fixtures.user_payloads[0], "new": fixtures.user_payloads[1],},
         )
+
+
+class SubscriptionTasksTest(TestCase):
+    def test_creating_and_deleting_subscriptions(self) -> None:
```

Comment:
I'd split that into multiple tests:
- Checking that author is subscribed to the thread
- Checking that commenters are subscribed to the thread
- Checking that deleting subscriptions indeed deletes the subscription